lib/templates/middleAssignment/main.dart

```diff
@@ -1,21 +1,84 @@
 import 'package:flutter/material.dart';
+import 'active_session_screen.dart';
 
 void main() {
-  runApp(const MiddleAssigmentApp());
+  runApp(MyApp());
 }
 
-class MiddleAssigmentApp extends StatelessWidget {
-  const MiddleAssigmentApp({super.key});
-
+class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Middle Assigment',
+      title: 'Your App Name',
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
-      // TODO - complete assignment
-      home: null,
+      home: HomeScreen(),
+    );
+  }
```

Comment:
Cannot run, not working project 
<img width="877" alt="Screenshot 2024-06-24 at 20 45 20" src="https://github.com/timur-harin/Sum24FlutterCourse/assets/64702874/0ff7e2ab-aa40-4be0-9cd9-d7de957f5bc3">